app/uk/gov/hmrc/incorporatedentityidentificationfrontend/repositories/JourneyConfigRepository.scala

```diff
@@ -23,7 +23,6 @@ import play.api.libs.json._
 import uk.gov.hmrc.incorporatedentityidentificationfrontend.config.AppConfig
```

Comment:
I noticed when updating sole trader there seems to be an issue with using the same name for the repository class and object. I suspect when you inject an instance of the repository into another class Scala 3 doesn't know whether to use the class or the object. I fixed the problem by changing the object name to JourneyConfigRepositoryHelper.

Your way seems to work too so I guess we can leave it as it is.